### DISCUSSION TITLE
Inconsistency in toggling visibility of "search/find bars" in Editor.

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#7076] [Editor]
In the Output panel, the "Filter Messages" bar can be toggled via CTRL + F (if that's the shortcut definition used), however, the "find bar" in both the ScriptEditor and EditorHelp are invoked via CTRL + F but must be closed via the ESC key.  Why not make them all consistent and open/close using only CTRL + F ?

**Comment by theraot** [comment]
By the way, notice that closing the filter in the Output panel does not remove the filter. *This is not a counter argument, I'm just pointing this out.*
